File: 2025046018_김래현/Week02/2609.c
#include <stdio.h>

int gcd(int x, int y) {
    int temp = 1;
    while (temp != 0) {
        temp = x%y;
        x=y;
        y=temp;
    }
    return x;
}
int lcm(int x, int y) {
    return x*y/gcd(x, y);
}

int main() {
    int a,b;
    scanf("%d%d", &a, &b);
    printf("%d\n%d", gcd(a, b), lcm(a, b));
}

File: 2021040034_김동균/Week02_Math/2609.py
import sys

def gcd(a,b):
    if(a<b):
        temp=a
        a=b
        b=temp
    if(b == 0):
        return a
    return gcd(b, a%b)
        
def lcm(a,b):
    if(a<b):
        temp=a
        a=b
        b=temp
    return int((a*b)/gcd(a,b))

input = sys.stdin.readline().rstrip()
a, b = map(int, input.split())
print(gcd(a,b))
print(lcm(a,b))